newsfragments/3457.feature

```diff
@@ -0,0 +1 @@
+Client nodes can now be introduced to storage nodes using an HTTP-based introducer.
```

Comment:
Mostly an "aside" here, but I'd like our "proposal process" to be able to merge a proposal in `docs/proposed` like this when we decide the **proposal** is good enough -- and then move it to `docs/` when actually implemented. I still have "write a proposal proposal down" in my TODO but if you agree, the above could say that we introduced a "proposal for an http-based introducer" or so, and then we could actually merge these proposal-docs while the implementation happens.